source/renderer/app/components/widgets/tooltips/QuestionMarkTooltip.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import SVGInline from 'react-svg-inline';
+import questionMarkIcon from '../../../assets/images/question-mark.inline.svg';
+import styles from './QuestionMarkTooltip.scss';
+
+type Props = {
+  key?: string,
+  content: string | Node,
```

Comment:
The `Node` type includes `string` so you don't need to specify it.

Reply:
Yeah i thought so too but Flow complained about it so that's why i added it (i think, will double check)

Reply:
Actually, is that the React `Node` type for sure? Shouldn't it be prefixed with `React.`?